gammapy/data/event_list.py

```diff
@@ -86,8 +87,9 @@ class EventList:
 
     """
 
-    def __init__(self, table):
+    def __init__(self, table=None, meta=None):
```

Comment:
Why the `table` can is None by default? There should not be a default argument here.

Reply:
Oops, a mistake on my behalf